pages/work.tsx

```diff
@@ -17,6 +18,10 @@ type Props = {
 const Work = ({ personalDetails, companyDetails }: Props): JSX.Element => {
   return (
     <>
+      <Head>
+        <title>Harsh Goel | Work</title>
+        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
```

Comment:
We won't need to add this everytime.

Reply:
Are you talking about `<meta>` tag? @harshgoel05

Reply:
yes  <meta name="viewport" content="initial-scale=1.0, width=device-width" /> sorry saw this now :) too late to the party